hooks/post_gen_project.py

```diff
@@ -27,6 +27,13 @@ def remove_dir(path):
         remove_dir(".github")
         remove_dir(".dependabot")
 
+    if "{{ cookiecutter.open_source_project }}".lower() == "n":
```

Comment:
Not sure if we need really this specific value, since we already state that `bitbucket` is non open source then. you can also make it part of the section above.

Reply:
It can be still OSS on bitbucket. And one may still want these files.

Reply:
Imagine you create a project with opensource in mind but you want first host it internally to for example review it? With this special flag you can treat the project still as opensource. Same applies for github? Imagine we would use GH enterprise and host a project in a private repo for internal use only?

Reply:
If that is the case then you would already add all necessary files to ensure full context get reviewed rather then get at the end confronted with new files.

Reply:
I would distinguish vcs host from content.
We just assume that Bitbucket is more for non-OSS and offer suitable default option.
The main option here is whether is it OSS or not. And content is defined by this option.